### ISSUE TITLE
Cancel Button doesn't show on Popup

### ISSUE THREAD

**Issue by kznestrela** [tschoffelen/react-native-map-link#114]
I have a Button with the function "showLocation" and whe I click it the Popup shows whitout the cancel button, so i need to open an application (like Google Maps app) and close it to get back to my app.

Here is the Button logic:
 ```
<Button color="#ffc500" onPress={() => showLocation({
          latitude: -23.557818,
          longitude: -46.668085,
          dialogTitle: 'Centro de Convenções Rebouças',
          dialogMessage: 'Qual app você gostaria de abrir?', 
          cancelText: 'Cancelar',
          alwaysIncludeGoogle: true,
        })} title="Abrir no mapa" />
```

And here is the result after clicking the button:
![issue maps-link](https://user-images.githubusercontent.com/53792747/70539810-eb700500-1b42-11ea-86d7-d5e3723c718c.jpeg)

Any solutions to show "cancel button"?

react-native version: 0.61.2
react-native-map-link version: 2.5.2

**Comment by tschoffelen** [comment]
Wouldn't you use the hardware back button on your device to do this usually? Sorry, not super familiar with Android, so might be wrong.

**Comment by kznestrela** [comment]
I expect that clicking out of the popup and the hardware back button closes the popup, but neither of it are working

**Comment by tschoffelen** [comment]
Resolved by #119, which has been released to NPM as `react-native-map-link@2.7.2`.
